scripts/apps/authoring/authoring/components/CharacterCount.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+import {gettext} from 'core/utils';
+import {cleanHtml} from '../helpers';
+
+interface IProps {
+    limit: number;
+    html: boolean;
+    item: string;
```

Comment:
I assume `item` is plain text version of `html` and only one can be passed? If so, it'd be better to make it more explicit by defining a more precise type.

```
interface IProps {
    limit: number;
    value: {text: string} | {html: string};
}
```